mods/railcraft/api/core/items/StackFilter.java

```diff
@@ -113,4 +115,15 @@ public boolean apply(ItemStack stack) {
         };
     }
 
+    public static StackFilter anyOf(@Nonnull final ItemStack[] stacks) {
+        return new StackFilter() {
```

Comment:
Yes. And its use case is for an inventory so even more awkward than it should be.